src/common/HTTPRequest.ts

```diff
@@ -459,6 +717,9 @@ const errorReasons: Record<ErrorCode, Protocol.Network.ErrorReason> = {
   failed: 'Failed',
 } as const;
 
+export type ActionResult = 'continue' | 'abort' | 'respond';
+export const DEFAULT_INTERCEPT_PRIORITY = 0;
```

Comment:
Earlier, we talked about this being 10 - what's the reason you changed this to 0?

Reply:
I can't think of a use case where someone would intentionally want a priority lower than the default, but I suppose it could happen.

Reply:
Sounds good!